.github/workflows/bumpversion.yml

```diff
@@ -7,9 +7,7 @@ on:
 
 jobs:
   bump-version:
-    if: |
```

Comment:
this was not working, the docs are a disaster:
https://github.com/orgs/community/discussions/25641

I don't actually have any guarantees that the new code will work. I believe it *was* working, but there's no way to know but to merge